obspy/io/seiscomp/sc3ml.py

```diff
@@ -128,9 +128,14 @@ def _read_sc3ml(path_or_file_object):
     """
     root = etree.parse(path_or_file_object).getroot()
 
-    # Fix the namespace as its not always the default namespace. Will need
-    # to be adjusted if the sc3ml format gets another revision!
-    namespace = "http://geofon.gfz-potsdam.de/ns/seiscomp3-schema/0.7"
+    # Code can be used for version 0.7, 0.8, and 0.9
+    basespace = "http://geofon.gfz-potsdam.de/ns/seiscomp3-schema"
+    for version in SCHEMA_VERSION:
+        namespace = "%s/%s" % (basespace, version)
+        if(root.find("{%s}%s" % (namespace, "Inventory"))) is not None:
```

Comment:
just nit picking.. but the enclosing brackets are not necessary, can you get rid of them and add a space after `if` (it's a builtin and not a function after all.. ;-) )? Thanks!